Src/BootCamp.Chapter/BalanceStats.cs

```diff
@@ -0,0 +1,258 @@
+﻿using System;
+using System.Globalization;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    public class BalanceStats
+    {
+        private const string NotAvailable = "N/A.";
+
+        /// <summary>
+        /// Return name and balance(current) of person who had the biggest historic balance.
+        /// </summary>
+        public string FindHighestBalanceEver(string[] peopleAndBalances)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make 'FindHighestBalanceEver' a static method.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5063788)